dev/com.ibm.ws.session.cache/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -30,6 +30,9 @@ properties.desc=List of vendor specific JCache configuration properties, which a
 extraProperties.name=Vendor specific properties
 extraProperties.description=List of vendor specific JCache configuration properties, which are passed to the JCache provider when obtaining the CacheManager.
 
+jCacheManagerRef=JCache CacheManager reference
+jCacheManagerRef.desc=The JCache CacheManager reference that will be used to get the HTTP session cache. Use of JCache results in the libraryRef, uri and properties configuration attributes being ignored as those properties are used from the referenced JCache CacheManager.
```

Comment:
suggested edit

The JCache CacheManager reference that is used to get the HTTP session cache. Use of JCache results in the libraryRef, uri, and properties configuration attributes are ignored because those properties are used from the referenced JCache CacheManager.

Reply:
Modified this a bit. I removed the `Use of ...` sentence and added `This setting is ignored when jCacheManagerRef is used.` to the three mentioned attributes.